webroot/src/store/license/license.actions.ts

```diff
@@ -13,18 +13,19 @@ export default {
     getLicenseesRequest: async ({ commit, getters, dispatch }, { params }: any) => {
         commit(MutationTypes.GET_LICENSEES_REQUEST);
 
-        // Temp for limited server paging support
         if (params?.getNextPage) {
             params.lastKey = getters.lastKey;
+        } else if (params?.getPrevPage) {
+            params.prevLastKey = getters.prevLastKey;
         }
 
-        await dataApi.getLicensees(params).then(async ({ lastKey, licensees }) => {
-            // Temp for limited server paging support
+        await dataApi.getLicensees(params).then(async ({ prevLastKey, lastKey, licensees }) => {
+            // Support for limited server paging support
             if (!licensees.length && params?.getNextPage) {
                 throw new Error('end of list');
```

Comment:
I assume this is the error that's thrown that's identified upstream by checking that `msg == 'end of list'`. Could we easily make a new Error type that's more specific?